<?xml version="1.0"?>
<robot name="dmu11">
  <link name="base_link">
    <visual>
    <origin  rpy="0 0 1.670" xyz="0 0 0" />
      <geometry>
        <mesh filename="package://dmu11_ros2_driver/meshes/dmu11.stl" scale="0.1 0.1 0.1"/>
      </geometry>
    </visual>
  </link>
</robot>
